lib/teleterm/api/protogen/js/v1/service_grpc_pb.d.ts

```diff
@@ -20,7 +20,8 @@ interface ITerminalServiceService extends grpc.ServiceDefinition<grpc.UntypedSer
     listLeafClusters: ITerminalServiceService_IListLeafClusters;
     listDatabases: ITerminalServiceService_IListDatabases;
     listDatabaseUsers: ITerminalServiceService_IListDatabaseUsers;
-    listServers: ITerminalServiceService_IListServers;
+    getAllServers: ITerminalServiceService_IGetAllServers;
+    getServers: ITerminalServiceService_IGetServers;
```

Comment:
This PR needs an equivalent PR in webapps which updates the JavaScript protos and the client, doesn't it? It'd need to be merged before this PR goes into master.

Reply:
PR for webapps [here](https://github.com/gravitational/webapps/pull/1164)